.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: python
 python:
-  - "3.5"
   - "3.6"
+  - "3.7"
 
 install:
   - pip install coveralls
+  - pip install black
   - pip install -r requirements.txt
 
 script:
+    - black . --line-length 120
```

Comment:
Replacing `pycodestyle` with `black` which also formats the code for you.
Running code style check before test as it completes faster.